src/sidebar/components/SearchInput.tsx

```diff
@@ -67,12 +67,6 @@ function useSearchKeyboardShortcuts(
 }
 
 export type SearchInputProps = {
-  /**
-   * When true, the input field is always shown. If false, the input field is
-   * only shown if the query is non-empty.
-   */
-  alwaysExpanded?: boolean;
```

Comment:
Is this not needed any more when the feature flag is off?

Reply:
I initially decided to unconditionally replace the component used in stream with the new one, so this prop was no longer used.

But I'm having second thoughts. It's pretty easy to check the feature flag in order to use one component or the other, so perhaps I roll back this change and make sure the feature flag allows for the new behavior to be completely disabled if desired.